packages/@aws-cdk/cdk/lib/resource-policy.ts

```diff
@@ -61,7 +63,7 @@ export interface ResourceSignal {
    * The timeout period starts after AWS CloudFormation starts creating the resource, and the timeout expires no sooner
    * than the time you specify but can occur shortly thereafter. The maximum time that you can specify is 12 hours.
    */
-  readonly timeout?: string;
+  readonly timeout?: Duration;
```

Comment:
Not sure we should change this... It's an "L1"

Reply:
Well yeah sure, but there is no L2 way to access this, and in any case, I feel like this is better specified in more semantic ways... I think thes may be the only use-case for the ISO-formatted duration, though...